docs/content/docs/dev/table/materialized-table/overview.md

```diff
@@ -0,0 +1,65 @@
+weight: 1
+type: docs
+aliases:
+- /dev/table/materialized-table.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Introduction
+
+Materialized tables are a new feature introduced in Flink 1.20 aimed at simplifying both batch and stream processing development processes, providing a consistent development experience. Users only need to specify the query and data freshness when creating a materialized table, and the framework will automatically choose the appropriate execution mode.
+
+{{< hint warning >}}
+This feature is in the MVP stage, only supported for use with `sql gateway` and in `Flink standalone` mode, and is currently not recommended for production use.
+{{< /hint >}}
+
+# Core Concepts
+
+Materialized tables encompass the following core concepts: Data Freshness, Refresh Mode, Query Definition, and Table Schema.
+
+## Data Freshness
+
+Data freshness measures the lag of the materialized table compared to the upstream table. After creating a materialized table, the flink framework will strive to meet the data freshness requirements, though it cannot guarantee complete compliance.
+
+Data freshness is a crucial attribute of a materialized table, serving two main purposes:
+1. **Determining the Refresh Mode**: Currently, there are CONTINUOUS and FULL modes. For details on how to determine the refresh mode, refer to the [materialized-table.refresh-mode.freshness-threshold]({{< ref "docs/dev/table/config" >}}#materialized-table-refresh-mode-freshness-threshold) configuration item.
+    - CONTINUOUS mode: Real-time jobs continuously refresh the materialized table.
+    - FULL mode: The workflow scheduler periodically triggers batch jobs to refresh the materialized table.
+2. **Determining the Refresh Frequency**:
+    - In CONTINUOUS mode, data freshness is converted into the `checkpoint` frequency of the background real-time job.
+    - In FULL mode, data freshness is converted into the scheduling cycle of the workflow.
+
+## Refresh Mode
+
+There are two refresh modes: FULL and CONTINUOUS. By default, the refresh mode is inferred based on data freshness. For specific business scenarios, users can explicitly specify the refresh mode, which will take precedence over the data freshness inference.
```

Comment:
There are two refresh modes: FULL and CONTINUOUS. By default, the refresh mode is inferred based on data freshness. Users can explicitly specify the refresh mode for specific business scenarios, which will take precedence over the data freshness inference.

- **CONTINUOUS Mode**: The Flink streaming job incrementally updates the materialized table data, with the data refresh frequency matching the job's `checkpoint` interval.
- **FULL Mode**: The scheduler periodically triggers a full update of the materialized table data, with the data refresh cycle matching the workflow's scheduling cycle.
    - The default overwrite behavior is at the table level. If there are partition fields and the time partition field format is specified via [partition.fields.#.date-formatter]({{< ref "docs/dev/table/config" >}}#partition-fields-date-formatter), the overwrite is by partition. Only the latest partition is refreshed each time.